examples/aqua-processor/README.md

```diff
@@ -156,6 +156,12 @@ Start IPOPP services:
 
 This completes the IPOPP installation and configuration. 
 
+### Level1 & Level2 products
```

Comment:
Do you have to call out enabling additional services in the IPOPP configuration UI?

Reply:
Good point, let me update that quickly. Thanks!

Reply:
I have included enabling the installed services in the UI, can you review it?